Sources/SwiftGen/Config/ConfigEntry.swift

```diff
@@ -38,40 +40,32 @@ extension Config {
 
 extension Config.Entry {
   init(yaml: [String: Any]) throws {
```

Comment:
Not sure if a change like this is worth doing in this current PR or not, but wondering if those `yaml:` labels were a good choice?

I mean, that configuration content could come in any form in the future, hell one might already use JSON if they wish, and maybe one day we would support any other format capable of serialising Hash+Arrays+Strings structures (Plist, whatever), all that to say that there's no real reason in those inits to name the label `yaml:` rather than anything else (I know, I personally use `init(json: )` constructors in my own projects when I know the data come from a REST/JSON web service, so guilty as charged here too, but worth opening the discussion anyway)

Nothing urgent, no impact on the behavior, and as I said could be an internal refactor later, but at least we could start discussing and wondering if it's worth it and find a new name, or too much nitpicking and bikeshedding ^^

Reply:
I usually use `data`, but that's even less descriptive 🤷‍♂️

Reply:
All our inits similar to this take dictionaries as parameters, right (never arrays, always dicts)? So maybe `init(dictionary: )` would fit better?

_In Foundation/CoreGraphics Apple uses `CGPoint(dictionaryRepresentation: )` and the likes, so at least `Config.Entry(dictionary: ))` would be inspired by/similar to those…_